content/community/community.mdx

```diff
@@ -21,6 +21,7 @@ EclipseSource bears no responsibility for the accuracy, legality or content of t
 
 :::
 
+- [JSON Forms Blazor integration](https://github.com/neuroglia-io/Neuroglia.Blazor.JsonForms) by JBBianchi
```

Comment:
Should we maybe reference the company name instead of the company name as the author, i.e. by Neuroglia SRL
After all, it is hosted in their Github org

Reply:
Yes please, you can mention Neuroglia SRL instead of my name, that would be great 🙏